winch/codegen/src/codegen/mod.rs

```diff
@@ -1022,6 +1026,93 @@ where
         );
     }
 
+    /// Emit series of instructions that check the current epoch against its
+    /// deadline.
+    pub fn emit_epoch_check(&mut self) {
+        let cont = self.masm.get_label();
```

Comment:
I think a one-line comment describing what this label is could also be helpful to code readers.